.github/workflows/run_unit_tests.yml

```
@@ -8,12 +8,12 @@ env:
   POETRY_VERSION: 1.2.2
 
 jobs:
-  run_tests:
-    name: Run tests
+  run_unit_tests:
+    name: Run unit tests
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-latest, windows-latest, ubuntu-latest]
+        os: [macos-latest, ubuntu-latest, windows-latest]
```

revert unnecessary change again

```suggestion
        os: [macos-latest, windows-latest, ubuntu-latest]
```